governance.md

```diff
@@ -1,149 +1,285 @@
+      with no objections after 1 week
+  - Expectations
+    - should subscribe to kubernetes-dev@googlegroups.com and kubernetes-dev-announce@googlegroups.com
+    - should read the [developer guide](contributors/devel/README.md)
+    - expected to be familiar with project organization, conventions, policies, etc.
+    - we [expect](contributors/devel/community-expectations.md) that
+      frequent contributors will assist in our code-review process and with project
+      maintenance
+  - Benefits
+    - added to kubernetes github org
+    - trusted enough to run tests on their PRs automatically
+    - can invoke `@k8s-bot ok to test` for other contributors
+    - can invoke issue label and assignment commands
+    - if they choose public membership, they get a badge on their github profile
+    - TBD: swag
+- **REVIEWER**:
+  - includes all requirements, expectations, and benefits of MEMBERS
+  - Requirements
+    - org member for at least 3 months
+    - authored and/or reviewed at least 20 merged non-trivial PRs, including at least 3 as the primary
+      reviewer
+    - familiar enough with some part of the codebase to be in an [OWNERS](contributors/devel/owners.md)
+      file `reviewers` list
+    - nominated by an APPROVER for that part of the codebase, with no objections from other
+      APPROVERS
+    - may be asked to become a REVIEWER as a precondition for accepting a large code contribution
+  - Expectations
+    - responsible for project quality control
+    - will be assigned related PRs and test bugs
+    - responsive to [code review](contributors/devel/collab.md) requests as per 
+      [community expectations](contributors/devel/community-expectations.md)
+  - Benefits
+    - can champion incubator repos
+    - added to the repo’s reviewers github team (e.g., [`kubernetes-reviewers`](https://github.com/orgs/kubernetes/teams/kubernetes-reviewers))
+    - gets "read access" to the repository
+    - gets a badge on PR and issue comments
+    - TBD: swag (e.g., ribbons at conferences, tshirts)
+- **APPROVER**:
+  - includes all requirements, expectations, and benefits of REVIEWERS
+  - Requirements
+    - previously a REVIEWER for that part of the codebase for at least 3 months
+    - authored and/or reviewed at least 30 merged non-trivial PRs, including at least 10 as the primary
+      reviewer
+    - familiar enough with some part of the codebase to be in an [OWNERS](contributors/devel/owners.md)
+      file `approvers` list
+    - demonstrated sound technical judgement
+    - nominated by a relevant APPROVER, with no objections from other APPROVERS in that area
+    - may be asked to become an APPROVER as a precondition for accepting a large code contribution
+  - Expectations
+    - responsive to [code review](contributors/devel/collab.md) requests as per
+      [community expectations](contributors/devel/community-expectations.md)
+    - mentor REVIEWERS and CONTRIBUTORS
+  - Benefits
+    - TBD: swag
+- **MAINTAINER**:
```

Comment:
I wonder if we could extract REVIEWER and APPROVER (and OWNERmaybe?) from the ladder, and still express their importance.  These specifically are about sub-trees, while the rest (seem to be) about the whole repo.  They are more like modifiers on MEMBER.

Would that help address the "too many rungs in the ladder" point some were making?